maestro/src/main/java/org/intocps/maestro/MableSpecificationGenerator.java

```diff
@@ -313,6 +314,15 @@ private ARootDocument generateFromDocuments(List<ARootDocument> documentList, In
             logger.info("\tImports {}", ("[ " + String.join(" , ", importedModuleNames) + " ]"));
 
 
+            // Extract Instance Mapping Statements (@map a -> x) and add to the unitrelationship
```

Comment:
sounds like this should be in its own method

Reply:
Fixed